specs/darwin/system_extensions.table

```diff
@@ -8,7 +8,8 @@ schema([
     Column("version", TEXT, "System extension version"),
     Column("category", TEXT, "System extension category"),
     Column("bundle_path", TEXT, "System extension bundle path"),
-    Column("team", TEXT, "Signing team ID")
+    Column("team", TEXT, "Signing team ID"),
+    Column("is_managed", INTEGER, "1 if managed by MDM system extension payload configuration, 0 otherwise")
```

Comment:
There's a good chance this is my own lack of knowledge, but I feel confused by terminology. `is_managed` implies this extension was put here by a management server. But I think it's telling us whether or not it's on a allow-list? (Which does not directly say anything about why the extension is installed)

Is that correct?

Reply:
You are correct. The management server only manages the configuration and not put the system extension to the system. I named the flag `is_managed` because it just shows the flag if the system extension is getting configured by the MDM server. 

I am not sure if I should change it to `allowed` because there is a payload flag to enable/disable user approval of system extensions. If a system extension is not running in lockdown mode, all system extension is allowed. Those which are not managed will require user approval.

Reply:
Hrm. Naming is always so hard. `mdm_allowed`? `mdm_allowlist`? Dunno?

Reply:
What about `mdm_managed`? I like anything but the "is_*" pattern, meaning I don't like having the word "is" included in column names.

Reply:
My dislike of `managed` is that it _strong_ implies the MDM has installed it. But the discussion here is that it's configured by the MDM, but installed by the end user. `mdm_configured`?